saleor/webhook/serializers.py

```diff
@@ -1,59 +1,114 @@
+        "base_price": quantize_price(base_price.amount, currency),
+        "currency": currency,
+        "full_name": variant.display_product(),
+        "product_name": product.name,
+        "variant_name": variant.name,
+        "attributes": serialize_product_or_variant_attributes(variant),
+        "product_metadata": line_info.product.metadata,
+        "product_type_metadata": line_info.product_type.metadata,
+    }
 
+
+def serialize_checkout_lines_with_taxes(
+    checkout_info: "CheckoutInfo",
+    manager: PluginsManager,
+    lines: Iterable["CheckoutLineInfo"],
+) -> List[dict]:
+    data = []
+    checkout = checkout_info.checkout
+    currency = checkout.currency
+
+    for line_info in lines:
+        unit_price_data = calculations.checkout_line_unit_price(
+            manager=manager,
+            checkout_info=checkout_info,
+            lines=lines,
+            checkout_line_info=line_info,
+            discounts=[],
+        )
+        unit_price = quantize_price(unit_price_data.price_with_sale, currency)
+        unit_price_with_discounts = quantize_price(
+            unit_price_data.price_with_discounts, currency
+        )
         data.append(
             {
-                "id": line_id,
-                "sku": variant.sku,
-                "variant_id": variant.get_global_id(),
-                "quantity": line_info.line.quantity,
-                "charge_taxes": product.charge_taxes,
-                "base_price": str(quantize_price(base_price.amount, currency)),
-                "price_net_amount": str(
-                    quantize_price(unit_price.net.amount, currency)
+                **_get_checkout_line_payload_data(checkout, line_info),
+                "price_net_amount": unit_price.net.amount,
+                "price_gross_amount": unit_price.gross.amount,
+                "price_with_discounts_net_amount": unit_price_with_discounts.net.amount,
+                "price_with_discounts_gross_amount": (
+                    unit_price_with_discounts.gross.amount
                 ),
-                "price_gross_amount": str(
-                    quantize_price(unit_price.gross.amount, currency)
-                ),
-                "currency": currency,
-                "full_name": variant.display_product(),
-                "product_name": product.name,
-                "variant_name": variant.name,
-                "attributes": serialize_product_or_variant_attributes(variant),
-                "product_metadata": line_info.product.metadata,
-                "product_type_metadata": line_info.product_type.metadata,
             }
         )
     return data
 
 
+def serialize_checkout_lines_without_taxes(
+    checkout: "Checkout",
+    lines: Iterable["CheckoutLineInfo"],
+    included_taxes_in_price: bool,
+) -> List[dict]:
+    def untaxed_price_amount(price: TaxedMoney) -> Decimal:
+        return quantize_price(
+            get_base_price(price, included_taxes_in_price), checkout.currency
```

Comment:
Method name says we are serializing lines without taxes but we have `included_taxes_in_price` to control taxes.

Reply:
I've changed the name from `included_taxes_in_price` to `use_gross_as_base_price` where appropriate